cypress/support/pages/signIn.pageObject.js

```diff
@@ -3,16 +3,19 @@ import PageObject from '../PageObject';
 class SignInPageObject extends PageObject {
   url = '/user/login';
 
-  get emailField() {
-    return cy.getByDataCy('email-sign-in');
+  emailField(email) {
```

Comment:
Consider renaming the method 'emailField' to 'typeEmail' for better readability and to follow the camelCase naming convention.